solar-land/scss/main.scss

```diff
@@ -0,0 +1,7 @@
+@import "variables"; // import variables file that contains color and font import
```

Comment:
I think the comment is not useful, everyone knows we define all those variables there. let's make code less and clean by removing redundant comments